packages/debugger/src/handlers/editor.ts

```diff
@@ -111,15 +180,50 @@ export class EditorHandler implements IDisposable {
       return;
     }
 
+    const editor = this._editor as CodeMirrorEditor;
+    editor.setOption('lineNumbers', true);
+    const breakpointGutter = [
+      this._breakpointState,
+      this._highlightState,
+      Prec.highest(
+        gutter({
+          class: 'cm-breakpoint-gutter',
+          renderEmptyElements: true,
+          markers: v => v.state.field(this._breakpointState),
+          initialSpacer: () => Private.breakpointMarker,
+          domEventHandlers: {
+            mousedown: (view, line): boolean => {
+              this._onGutterClick(view, line.from);
+              return true;
+            }
+          }
+        })
+      )
+    ];
+    editor.injectExtension(this._gutter.of(breakpointGutter));
+
+    this._uglyCM6Workaround();
     this._addBreakpointsToEditor();
+  }
 
+  // There is currently a bug in the gutter of CM6, gutter
+  // elements without any marker don't get a DOM class assigned.
+  // The bug is fixed in the master branch, we can remove this
+  // when it is released.
+  private _uglyCM6Workaround(): void {
```

Comment:
Could you add a link to the PR or a mention of the version that will contain the bug fix

Reply:
The fix has been released, this code is not needed anymore, I will remove it.